src/jdk.attach/share/classes/sun/tools/attach/HotSpotVirtualMachine.java

```diff
@@ -366,6 +369,87 @@ String readErrorMessage(InputStream in) throws IOException {
         return message.toString();
     }
 
+    void processCompletionStatus(IOException ioe, String cmd, InputStream sis) throws AgentLoadException, IOException {
```

Comment:
I was thinking more about describing what the parameters are - in particular it is unclear why the actual IOE can be replaced by a passed in one. If we wanted a custom message then I would at least expect to chain the actual IOE to the replacement.

Reply:
> I was thinking more about describing what the parameters are - in particular it is unclear why the actual IOE can be replaced by a passed in one. If we wanted a custom message then I would at least expect to chain the actual IOE to the replacement.

Done. Parameter ioe means if we get IOE during previous command execution, delay throwing it until command execution completion status has been read. Since all other parameters are straightforward and this is a private method, I don't write  strandard `@param` descriptions for them.